docs/runtime-compliance-testing.md

```diff
@@ -6,4 +6,375 @@ In order to be tested for [compliance][], runtimes MUST support at least one of
 
 * Version 1.0.0 of the [OCI Runtime Command Line Interface](command-line-interface.md).
 
-[compliance]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/spec.md#notational-conventions
+## Running the runtime validation suite
+
+The runtime validation suite uses [node-tap][], which is packaged for some distributions (for example, it is in [Debian's `node-tap` package][debian-node-tap]).
+If your distribution does not package node-tap, you can install [npm][] (for example, from [Gentoo's `nodejs` package][gentoo-nodejs]) and use it:
+
+```console
+$ npm install tap
+```
+### From a release
+
+Check if your release has pre-compiled tests on the [release page][releases] page.
+
+```
+$ tar xf runtime-tools-v0.6.0.tar.gz
+$ RUNTIME=runc tap ./runtime-tools-v0.6.0/validation/*.t
+```
+
+### From source
+
+Build the validation executables:
+
+```console
+$ make runtimetest validation-executables
+```
+
+Runtime validation currently [only supports](docs/runtime-compliance-testing.md) the [OCI Runtime Command Line Interface](doc/command-line-interface.md).
+If we add support for alternative APIs in the future, runtime validation will gain an option to select the desired runtime API.
+For the command line interface, the `RUNTIME` option selects the runtime command (`funC` in the [OCI Runtime Command Line Interface](doc/command-line-interface.md)).
+
+```
+$ sudo TAP="$(which tap)" RUNTIME=runc make localvalidation
```

Comment:
Why `TAP="$(which tap)"`?  If the issue is that you have `tap` in your `PATH` but not in the root user's `PATH`, we should explain that in the text.

And `which` is not in POSIX; it's more portable to use [`command`][1] (e.g. `TAP=$(command -v tap)`).

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html